engine/resources/exec.go

```diff
@@ -29,6 +29,8 @@ import (
 	"syscall"
 	"time"
 
+	"golang.org/x/sys/unix"
```

Comment:
Imports go in three groups: with the middle group being our internal code... This needs to go in the third at the end. Here and elsewhere...

https://github.com/golang/go/wiki/CodeReviewComments#imports